relayer/relays/beacon/state/beacon.go

```diff
@@ -196,7 +196,8 @@ type BlockRootsContainer interface {
 type BeaconBlock interface {
 	UnmarshalSSZ(buf []byte) error
 	GetBeaconSlot() uint64
-	GetExecutionPayload() *ExecutionPayloadCapella
+	CapallaExecutionPayload() *ExecutionPayloadCapella
```

Comment:
Naming should be consistent

```suggestion
	ExecutionPayloadCapella() *ExecutionPayloadCapella
```

Reply:
https://github.com/Snowfork/snowbridge/pull/981/commits/d4bc62bc6f4ce11f379441d7f7f913dbea7fc773